app/components/channel_item/channel_item.tsx

```diff
@@ -120,13 +125,15 @@ const ChannelItem = ({
     isOnCenterBg = false,
     showChannelName = false,
     isOnHome = false,
+    urgentMentionCount = 0,
 }: Props) => {
     const {formatMessage} = useIntl();
     const theme = useTheme();
     const isTablet = useIsTablet();
     const styles = getStyleSheet(theme);
 
     const channelName = (showChannelName && !isDMorGM(channel)) ? channel.name : '';
+    const hasUrgent = urgentMentionCount > 0;
```

Comment:
I think I remember that you mention something about mentions already including the urgent mentions, is that correct? maybe just add a comment here that makes it clear

Reply:
Yup, this was the comment: https://github.com/mattermost/mattermost-mobile/pull/8801#discussion_r2061294431, will add the comment as well.